### DISCUSSION TITLE
Docker Overlay support?

### DISCUSSION THREAD

**Discussion by mrstux** [Jip-Hop/jailmaker#207] [General]
Bumped into this,

https://theorangeone.net/posts/docker-lxc-storage/

Wondered if it applied to jailmaker. 

Didn’t see overlayfs in the docker config

Thoughts?

**Comment by Jip-Hop** [comment]
Try running `docker info` in the jail.

**Comment by mrstux** [comment]
```
Storage Driver: overlay2
  Backing Filesystem: zfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
```

That's good then :)

Gottit, Jailmaker > LXC 

:)

**Reply by Jip-Hop** [reply]
I think you'd see the same on lxc these days. Support for native overlay diff is thanks to recent kernel improvements.
